var/spack/repos/builtin/packages/py-elephant/package.py

```diff
@@ -37,9 +39,9 @@ class PyElephant(PythonPackage):
     depends_on("python@3.7:", type=("build", "run"), when="@0.11.0:")
```

Comment:
Missing dep on Python 3.8+

Reply:
added dep for python 3.8 + starting from elephant 0.12.0